lib/actions/ssh-keys.ts

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2019 Balena Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CommandDefinition } from 'capitano';
+import { stripIndent } from 'common-tags';
+
+export const sshKeys: CommandDefinition<
+	{
+		file?: string;
+		title?: string;
+	},
+	{
+		'list-fingerprints'?: boolean;
+		'list-keys'?: boolean;
+		'delete-key'?: number;
+		'delete-all-keys'?: boolean;
+		'add-all-keys'?: boolean;
+	}
+> = {
+	signature: 'ssh-keys [file] [title]',
+	primary: true,
+	description: 'Manage ssh keys',
+	help: stripIndent`
+		This command exposes an interface similar to \`ssh-add\` and can be used to manage your authorized keys.
+
+		Examples:
+
+			$ balena ssh-keys <file> [title]
+			$ balena ssh-keys -L
+			$ balena ssh-keys -d <key-id>
+			$ balena ssh-keys -D
+			$ balena ssh-keys -A
+	`,
+	options: [
+		{
+			signature: 'list-keys',
+			alias: 'L',
+			description: 'List public key parameters of all identities.',
+			boolean: true,
+		},
+		{
+			signature: 'delete-key',
+			alias: 'd',
+			description: 'Delete identity.',
+			parameter: 'key-id',
+		},
+		{
+			signature: 'delete-all-keys',
+			alias: 'D',
+			description: 'Delete all identities.',
+			boolean: true,
+		},
+		{
+			signature: 'add-all-keys',
+			alias: 'A',
+			description: 'Add all identities stored at ~/.ssh/id_{ed25519,{r,d}sa}',
```

Comment:
these would be the private keys... it should be `~/.ssh/id_{ed25519,{r,d}sa}.pub` as per the implementation 👍